plasma-roundup/draft-plasma-article.md

```diff
@@ -0,0 +1,112 @@
+
+## FourthState Labs
+
+Among the early Plasma implementations was [FourthState Labs](https://github.com/FourthState/plasma-mvp-rootchain), whose design included a rootchain contract according to the Plasma MVP. This rootchain, which other projects have also incorporated, is a series of Solidity smart contracts. It may be a good place to start if you are considering building your own Plasma chain. It is designed to maintain a mapping from from block number to merkle root, processing deposits, withdrawals and resolving transaction disputes. FourthState has written tests for these features which you can run by following these steps:
+
+```shell
+git clone https://github.com/fourthstate/plasma-mvp-rootchain
+cd plasma-mvp-rootchain
+npm install
+ganache-cli // In seperate terminal window
+npm test
+```
+
+![FourthState tests](/images-for-article/Fourth-Estate/fourth-estate.png)
+
+<!-- TODO: Running? -->
+
+## OmiseGO
+
+Other notable early MVP's include [OmiseGO's implementation](https://github.com/omisego/plasma-contracts).  OmiseGO aspires to enable financial inclusion and interoperability through a public, decentralized OMG network. A key component of this network is Plasma. OMG’s implementation which has a root chain, child chain and a client to interact with the Plasma chain, is different from the MVP specification, however. For instance, OMG has added protection against the threat of chain re-organization (which can result from 51% attacks). And, among other additions, it has built in support for ERC20 token handling.
+
+## Kyokan Plasma
+
+Making the significant jump from MVP to mainnet is [Kyokan](https://github.com/kyokan/plasma). Kyokan is a Golang implementation [extending the original MVP specification](https://kauri.io/article/7f9e1c04f3964016806becc33003bdf3/v4/minimum-viable-plasma-the-kyokan-implementation). Their architecture uses the FourthState rootchain contract reviewed above and also includes root nodes to process transactions and package them into blocks, broadcast blocks to validator nodes, process exits and more. The team has been working steady since March 2018 to build a production-ready plasma implementation. Last month, just one year later, Kyokan announced that they have achieved [two critical milestones on the way to that goal](https://medium.com/kyokan-llc/announcing-our-plasma-mvp-alpha-23a8bc9673fc): the launch of their MVP mainnet alpha (capable of an initial 1,000 transactions per second) and the completion of their security audit. To help test how their Plasma will behave now that they have released it into the wild, Kyokan has launched a game of Capture the Flag where people are invited to break the system and keep the funds “just let us know how you did it, so that the rest of the community can benefit.” Successful denial of service attacks and attacks that lead to a loss of funds in [the smart contract](https://etherscan.io/address/0x0cdd78c34a4305234898864c1daccdbb326a520d) will also be paid $1,000.
```

Comment:
```suggestion
Making the significant jump from MVP to mainnet is [Kyokan](https://github.com/kyokan/plasma). Kyokan is a Golang implementation [extending the original MVP specification](https://kauri.io/article/7f9e1c04f3964016806becc33003bdf3/v4/minimum-viable-plasma-the-kyokan-implementation). Their architecture uses the FourthState rootchain contract reviewed above and also includes root nodes to process transactions and package them into blocks, broadcast blocks to validator nodes, process exits and more. The team has been working steady since March 2018 to build a production-ready plasma implementation. Last month, Kyokan announced that they have achieved [two critical milestones on the way to that goal](https://medium.com/kyokan-llc/announcing-our-plasma-mvp-alpha-23a8bc9673fc): the launch of their MVP mainnet alpha (capable of an initial 1,000 transactions per second) and the completion of their security audit. To help test how their implementation will behave now that they have released it into the wild, Kyokan has launched a game of Capture the Flag where people are invited to break the system and keep the funds, “just let us know how you did it, so that the rest of the community can benefit.” Successful denial of service attacks and attacks that lead to a loss of funds in [the smart contract](https://etherscan.io/address/0x0cdd78c34a4305234898864c1daccdbb326a520d) will also be paid $1,000.
```